kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -304,22 +304,19 @@ define([
                         case JOB_REQUESTS.LOGS:
                             this.sendBusMessage(JOB, jobId, RESPONSES.LOGS, {
                                 jobId,
-                                error: msgData.message,
-                            });
-                            this.sendBusMessage(JOB, jobId, 'job-log-deleted', {
-                                jobId,
-                                message: msgData.message,
```

Comment:
bye bye `job-log-deleted` message type! Now the message is delivered as a `job-logs` message with an `error` key

Reply:
👋